src/_layouts/home.html

```diff
@@ -3,24 +3,28 @@
         <title>{{ page.title }}</title>
         <link rel="stylesheet" href="/assets/css/styles.css" />
         <link rel="stylesheet" href="/assets/css/splash.css" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
     </head>
     <body>
         <header class="header">
             <a href="#logo"><p>LIGHTNING HACKS</p></a>
         </header>
         <div class="splash" id="logo">
             <div class="splash--item">
-                {% include logo_yellow.html id="logo" animate=true width='50vw' %}
+                {% include logo_yellow.html id="logo" animate=true width='100%' %}
             </div>
             <div class="splash--item">
-                <div class="title"> 
-                    <p>Uma</p>
-                    <h3>INICIATIVA</h3>
-                    <p>de <b>hackers</b> para <b>hackers</b></p>
+                <div class="title">
```

Comment:
Maybe we should move this title to the homepage as well? and leave only the header in this layout? :thinking: